backend/app/services/chat/config/shell_checker.py

```
@@ -160,3 +160,81 @@ def get_team_first_bot_shell_type(db: Session, team: Kind) -> str:
+    if not bot_crd.spec or not bot_crd.spec.modelRef:
+        return False
+
+    # Get model from bot's modelRef
+    model = (
+        db.query(Kind)
+        .filter(
+            Kind.user_id == team.user_id,
+            Kind.kind == "Model",
+            Kind.name == bot_crd.spec.modelRef.name,
+            Kind.namespace == bot_crd.spec.modelRef.namespace,
+            Kind.is_active == True,
```

Ruff (0.14.14)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)